pkg/controller/controller_test.go

```diff
@@ -1131,6 +1160,11 @@ func TestReconcileBindingWithParameters(t *testing.T) {
 
 	testController.reconcileBinding(binding)
 
+	ns, _ := fakeKubeClient.Core().Namespaces().Get(binding.ObjectMeta.Namespace)
+	if string(ns.UID) != fakeBrokerClient.Bindings[instanceGUID+":"+bindingGUID].AppID {
```

Comment:
Use the BindingsMapKey method instead of constructing key yourself in case things change: fakebrokerapi.BindingsMapKey(instanceGUID, bindingGUID)